packages/adapters/subgraph/src/lib/helpers/parse.ts

```diff
@@ -73,6 +73,11 @@ export const originTransfer = (entity: any, asset: Record<string, AssetId>): Ori
     for (const relayerFee of entity.relayerFees) {
       relayerFees[relayerFee.asset] = relayerFee.fee;
     }
+  } else {
+    // TODO: Remove after relayer fee upgrade is completed on mainnet
```

Comment:
really this is after all the routers upgrade

Reply:
True. Updated the TODO.